piksi_multi_cpp/src/sbp_callback_handler/position_sampler.cc

```diff
@@ -207,9 +209,60 @@ bool PositionSampler::savePositionToFile(const Eigen::Vector3d& x,
     if (!fs::create_directories(path.parent_path())) return false;
   }
 
+  // Convert to common coordinate frames.
+  std::optional<Eigen::Vector3d> x_wgs84;
+  if (geotf_handler_.get()) {
+    Eigen::Vector3d x_wgs84_temp;
+    if (geotf_handler_->getGeoTf().convert("ecef", x, "wgs84", &x_wgs84_temp))
+      x_wgs84 = std::make_optional(x_wgs84_temp);
+  }
+
+  std::optional<Eigen::Vector3d> x_enu;
+  if (geotf_handler_.get()) {
+    Eigen::Vector3d x_enu_temp;
+    if (geotf_handler_->getGeoTf().convert("ecef", x, "enu", &x_enu_temp))
```

Comment:
is at this time of execution enu always defined? (could well be the case, just not sure ;-)

Reply:
Added the option to set ENU origin before saving to file.